Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj

```diff
@@ -45,7 +45,7 @@
     <Debugger>Xamarin</Debugger>
     <DevInstrumentationEnabled>True</DevInstrumentationEnabled>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <AndroidSupportedAbis>armeabi-v7a</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
```

Comment:
Is adding `x86` intentional?

Reply:
Yea in trying to speed up master this got removed so deploying to android device on windows needs it